nirc_ehr/resources/schemas/dbscripts/postgresql/nirc_ehr-21.002-21.003.sql

```diff
@@ -0,0 +1,3 @@
+SELECT core.executeJavaUpgradeCode('reloadStudy');
+SELECT core.executeJavaUpgradeCode('etl;{NIRC_EHR}/project');
```

Comment:
This is fine to leave here, but until we have a mechanism to reload the extensible table columns they'll still need to rerun the ETL after manually creating the columns

Reply:
I had it in my list to remove the extensible defn for project that I missed as now I am not pulling those columns from the different table after getting clarification yesterday. I think you caught 'isActive' column as well in extensible defn, I have removed the whole defn for project so need not to worry about the reload the defn in this commit but def a good thing for future auto re-load lists.